packages/remix-deno/server.ts

```diff
@@ -12,7 +12,7 @@ function defaultCacheControl(url: URL, assetsPublicPath = "/build/") {
 }
 
 export function createRequestHandler<
-  Context extends AppLoadContext | undefined = undefined,
+  Context extends AppLoadContext | undefined = undefined
```

Comment:
No real changes in this file, so I think we can just revert this one, to have a smaller git diff?

```sh
git checkout dev ./packages/remix-deno/server.ts
```